packages/backend/src/messages.ts

```diff
@@ -1,5 +1,5 @@
 const selectGeneratorQuestionHint =
-  "Select the generator that best fits the type of application you want to develop";
+  "When you create a project using a template, you become the code owner and as such, you are responsible for any updates or fixes that must be performed for it. We recommend you follow the industry best practice to use automated vulnerability scanning in the CI/CD pipeline to avoid supply chain and other cyber-attacks.";
```

Comment:
We recommend you to follow the software security industry best practice, to use automated vulnerability scanning in the CI/CD pipeline to avoid supply chain attack and other cyber-attacks.";

Reply:
To remove the first sentence or only to fix the second?